tests/2018-03-01/Microsoft.Insights.metricAlerts.tests.json

```diff
@@ -0,0 +1,108 @@
+{
+  "tests": [
```

Comment:
>tests [](start = 3, length = 5)

can you add one positive test case?

Reply:
First one is a positive test case. It should successfully create a metric alert.
@TianoMS